test/ao/sparsity/test_sparsifier.py

```diff
@@ -220,3 +240,57 @@ def test_mask_squash(self):
             module = g['module']
             assert not is_parametrized(module, 'weight')
             assert not hasattr(module, 'mask')
+
+    def test_sparsity_levels(self):
```

Comment:
optional: might be good to have at least one test case which tests the behavior directly (hardcode a tensor, apply m-n sparsity, verify individual values).  The current coverage is good but might not catch bugs with sorting, etc.